sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -150,6 +157,38 @@ export class Sender {
     sendBatch(messages: SendableMessageInfo[]): Promise<void>;
     }
 
+// @public
+export class ServiceBusAtomManagementClient extends ServiceClient {
+    constructor(connectionString: string, proxySettings?: ProxySettings);
+    // Warning: (ae-forgotten-export) The symbol "QueueOptions" needs to be exported by the entry point index.d.ts
```

Comment:
Review the warnings for the `ServiceBusAtomManagementClient` here. We would need the options to exported from the package